kura/org.eclipse.kura.web2/src/main/webapp/joint.shapes.devs.js

```diff
@@ -0,0 +1,169 @@
+joint.shapes.devs = {};
```

Comment:
@ctron This is part of JointJS library which is already been approved previously if I am not wrong. I just made some changes to it to cope up with wires.